besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -583,6 +583,14 @@ void setBannedNodeIds(final List<String> values) {
       arity = "1")
   private final File rpcHttpAuthenticationPublicKeyFile = null;
 
+  @SuppressWarnings({"FieldCanBeFinal", "FieldMayBeFinal"}) // PicoCLI requires non-final Strings.
+  @Option(
+      names = {"--rpc-http-authentication-jwt-algorithm"},
+      description =
+          "JWT encryption algorithm used by the public key - must be one of RS256, RS384, RS512, ES256, ES384 and ES512",
```

Comment:
I think this would be better done with the ${COMPLETION-CANDIDATES} magic. See NatMethod class

Reply:
Also s/ -/.
to match other options help

Reply:
maybe "Encryption algorithm used for the JWT public key. Possible values are ..." ?

Reply:
Can you elaborate on this one with the completion candidates comment? Not sure what you meant - also confused by the NatMethod class

Reply:
e4383ca